odl/tomo/geometry/conebeam.py

```diff
@@ -50,7 +50,8 @@ class HelicalConeFlatGeometry(DivergentBeamGeometry, AxisOrientedGeometry):
     detector positions.
     """
 
-    def __init__(self, angle_intvl, dparams, src_radius, det_radius, pitch,
+    def __init__(self, angle_intvl, dparams, src_radius, det_radius,
+                 pitch, pitch_offset=0,
                  agrid=None, dgrid=None, axis=[0, 0, 1], src_to_det=None,
                  detector_axes=None):
```

Comment:
Make `pitch_offset` and `detector_axes` part of `kwargs`, too many arguments here (even w/o the intervals)